core/src/modules/ops/group.ts

```diff
@@ -197,9 +199,10 @@ export namespace GroupOps {
       "ready"
     );
 
-    where["createdAt"] = { [Op.and]: [{ [Op.lt]: run.createdAt }] };
+    where.createdAt = { [Op.and]: [{ [Op.lt]: run.createdAt }] };
     if (highWaterMark) {
-      where["createdAt"][Op.and].push({ [Op.gte]: highWaterMark });
+      //@ts-ignore
+      where.createdAt[Op.and].push({ [Op.gte]: highWaterMark });
```

Comment:
This is a shame:

> Type 'unique symbol' cannot be used as an index type.ts(2538)
https://github.com/Microsoft/TypeScript/issues/24587

Reply:
https://tenor.com/view/curse-aukerman-comedy-fist-in-air-shaking-gif-5620082